src/resmom/linux/mom_mach.c

```diff
@@ -4537,46 +4542,32 @@ proc_get_btime(void)
 	return;
 }
 
-static char	stat_str[] =
-	"%d "		/* 1  pid %d The process id */
-"(%[^)]) "	/* 2  comm %s The filename of the executable */
-"%c "		/* 3  state %c "RSDZTW" */
-"%d "		/* 4  ppid %d The PID of the parent */
-"%d "		/* 5  pgrp %d The process group ID */
-"%d "		/* 6  session %d The session ID */
-"%*d "		/* 7  tty_nr %d The tty */
-"%*d "		/* 8  tpgid %d The process group that owns the tty */
-"%lu "		/* 9  flags %lu The flags of the process */
-"%*lu "		/* 10 minflt %lu minor faults */
-"%*lu "		/* 11 cminflt %lu minor faults proc + children */
-"%*lu "		/* 12 majflt %lu major faults */
-"%*lu "		/* 13 cmajflt %lu major faults proc + children */
-"%lu "		/* 14 utime %lu jiffies user mode */
-"%lu "		/* 15 stime %lu jiffies kernel mode */
-"%lu "		/* 16 cutime %ld jiffies proc + children user mode */
-"%lu "		/* 17 cstime %ld jiffies proc + children kernel mode */
-"%*ld "		/* 18 priority %ld nice value + 15 */
-"%*ld "		/* 19 nice %ld nice value */
-"%*ld "		/* 20 hard coded to 0 */
-"%*ld "		/* 21 itrealvalue %ld jiffies before next SIGALRM */
-"%lu "		/* 22 starttime %lu jiffies proc started after boot */
-"%lu "		/* 23 vsize %lu Virtual memory size in bytes */
-"%lu "		/* 24 rss %ld Resident Set Size */
-"%*lu "		/* 25 rlim %lu rss limit in bytes */
-"%*lu "		/* 26 startcode %lu program text start */
-"%*lu "		/* 27 endcode %lu program text end */
-"%*lu "		/* 28 startstack %lu start of stack */
-"%*lu "		/* 29 kstkesp %lu current stack pointer */
-"%*lu "		/* 30 kstkeip %lu current instruction pointer */
-"%*lu "		/* 31 signal %lu bitmap of pending signals */
-"%*lu "		/* 32 blocked %lu bitmap of blocked signals */
-"%*lu "		/* 33 sigignore %lu bitmap of ignored signals */
-"%*lu "		/* 34 sigcatch %lu bitmap of catched signals */
-"%*lu "		/* 35 wchan %lu wait channel */
-"%*lu "		/* 36 nswap %lu pages swapped */
-"%*lu "		/* 37 cnswap %lu Cumulative nswap for child procs */
-"%*d "		/* 38 exit_signal %d Signal to be sent to parent */
-"%*d";		/* 39 processor %d CPU number last executed on */
+static char	stat_str_pre[] =
+"%%d "		/* 1  pid %d The process id */
+"(%%[^)]) "	/* 2  comm %s The filename of the executable */
+"%%c "		/* 3  state %c "RSDZTW" */
+"%%d "		/* 4  ppid %d The PID of the parent */
+"%%d "		/* 5  pgrp %d The process group ID */
+"%%d "		/* 6  session %d The session ID */
+"%%*d "		/* 7  ignored:  tty_nr */
+"%%*d "		/* 8  ignored:  tpgid */
```

Comment:
GtiHub diff is lying.  Using a real editor one can see that the code is correct.